R/xml_find.R

```diff
@@ -203,3 +207,13 @@ xml_find_lgl.xml_nodeset <- function(x, xpath, ns = xml_ns(x)) {
 xml_find_lgl.xml_missing <- function(x, xpath, ns = xml_ns(x)) {
    logical(0)
 }
+
+# Deprecated functions ----------------------------------------------------
+
+#' @rdname xml_find_all
+#' @usage NULL
+#' @export
+xml_find_one <- function(x, xpath, ns = xml_ns(x)) {
+  .Deprecated("xml_find_first")
```

Comment:
The only downside of formally deprecating (i.e. instead of throwing a message) is that this will possible create new warnings for revdeps, so we'll need to do a release announcement at least two weeks prior to release. We probably don't need to do that otherwise because we're only adding new features.